05_distributed.ipynb

```diff
@@ -21,32 +21,32 @@
     "As we have seen so far, Dask allows you to simply construct graphs of tasks with dependencies, as well as have graphs created automatically for you using functional, Numpy or Pandas syntax on data collections. None of this would be very useful, if there weren't also a way to execute these graphs, in a parallel and memory-aware way. So far we have been calling `thing.compute()` or `dask.compute(thing)` without worrying what this entails. Now we will discuss the options available for that execution, and in particular, the distributed scheduler, which comes with additional functionality.\n",
```

Comment:
<p>This diff cell ends up looking wrong - worth checking how it renders.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-tutorial/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Not sure how ReviewNB is rendering the content. It looks normal in JupyterLab, as seen below.

<img src="https://user-images.githubusercontent.com/17503044/100690801-51a5f700-3355-11eb-8486-f72698662812.png" alt="drawing" width="500"/>